ui/pages/confirmations/components/confirm/info/approve/approve-static-simulation/approve-static-simulation.tsx

```diff
@@ -65,33 +66,38 @@ export const ApproveStaticSimulation = () => {
   );
 
   const simulationElements = (
-    <Box display={Display.Flex}>
-      <Box
-        display={Display.Inline}
-        marginInlineEnd={1}
-        minWidth={BlockSize.Zero}
-      >
-        {spendingCap === SPENDING_CAP_UNLIMITED_MSG ? (
-          <Tooltip title={formattedSpendingCap}>{formattedTokenText}</Tooltip>
-        ) : (
-          formattedTokenText
-        )}
+    <ConfirmInfoRow
+      label={t(isNFT ? 'simulationApproveHeading' : 'spendingCap')}
```

Comment:
Concerning `isNFT`, did we ever update the dependency to just `to` to prevent the repeated requests and re-renders?

Reply:
Yes, this is done on [this other PR](https://github.com/MetaMask/metamask-extension/pull/26906/files#diff-0208cae96ea3368b7cf5a2ecf8382f9d5f8e1356806be22f845a163cb8c25bc8R13) that will be merged first.

Reply:
I added this to this PR as well in any case.